js/backup_restore.js

```diff
@@ -636,7 +636,7 @@ function configuration_restore(callback) {
     function configuration_upload(configuration, callback) {
         function upload() {
             var activeProfile = null,
-                profilesN = 3;
```

Comment:
I think we need to have this version gated dependent on the firmware version. 3.0.0 is back up to 3 profiles. Or better, we should make the code able to handle the situation where 3 profiles are in a backup, but the current version of the firmware only supports 2.